.github/workflows/node.js.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: npm test
+      uses: actions/setup-node@v1
+    - run: sudo apt-get install -y libunbound-dev
+    - run: npm ci
+    - run: npm run build --if-present
+    - run: npm install -g nyc bmocha
+    - run: nyc --reporter lcov bmocha -- --reporter spec test/*.js
+
+    - name: eslint
+      uses: getsentry/action-eslint-fix@v1
```

Comment:
no `run` statement? I do kinda like when CI catches lint errors from contributors, it can help identify bugs quickly and keeps the codebase style consistent.

Reply:
Don't need a run statement as it is a plugin.

Reply:
Must mention I had an issue with this plugin and ended up switching to `hallee/eslint-action@1.0.3`

Reply:
You can try the changes locally by pushing this branch to your fork.

Reply:
> no `run` statement? I do kinda like when CI catches lint errors from contributors, it can help identify bugs quickly and keeps the codebase style consistent.

See here for an example of how linter can do annotations:

https://github.com/tuxcanfly/hsd/runs/927111110?check_suite_focus=true

Reply:
Decided to drop annotations for now. Running simple `eslint`.